simple_qa_rnn/.gitignore

```diff
@@ -1,4 +1,8 @@
-data/
+data/*
+data/glove/
 resources/
 saved_checkpoints/
 __pycache__/
+relation_prediction/*cache*/*
+.idea/*
```

Comment:
don't know why the gitignore in root directory does not catch the DS_Store files - let's keep it there.